src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
What will be use case for netstandard2.0?
There is no other supported .NET (Core/Framweork) versions which cannot reference eithe net6.0 or net462.
@srprash, can it be dropped?

Reply:
Does this change need a major version bump? Will it make this project incompatible for someone using netcore3.1?

Reply:
I think that we agreed to rename package (remove Contrib infix). It is the same level of breaking changes.

It will drop support for .NET Core 3.1